library/swift/test/BridgeTests.swift

```diff
@@ -0,0 +1,100 @@
+import Envoy
+import EnvoyEngine
+import Foundation
+import XCTest
+
+private let kMockTemplate = """
+mock_template:
+- name: mock
+  stats_domain: {{ stats_domain }}
+  device_os: {{ device_os }}
+  connect_timeout: {{ connect_timeout_seconds }}s
+  dns_refresh_rate: {{ dns_refresh_rate_seconds }}s
+  dns_failure_refresh_rate:
+    base_interval: {{ dns_failure_refresh_rate_seconds_base }}s
+    max_interval: {{ dns_failure_refresh_rate_seconds_max }}s
+  platform_filter_chain: {{ platform_filter_chain }}
+  stats_flush_interval: {{ stats_flush_interval_seconds }}s
+  app_version: {{ app_version }}
+  app_id: {{ app_id }}
+  virtual_clusters: {{ virtual_clusters }}
+"""
+
+private struct TestFilter: Filter {}
+
+final class BridgeTests: XCTestCase {
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  func testInitial() throws {
```

Comment:
We should add some more color to what this is actually testing (test names can be quite verbose which is fine)